libbrowser/src/libbrowser_cef.cpp

```diff
@@ -1323,16 +1330,28 @@ void MCCefBrowserBase::SetOverflowHidden(bool p_hidden)
 	MCCStringFree(t_overflow_script);
 }
 
-bool MCCefBrowserBase::GetScrollbars(void)
+bool MCCefBrowserBase::GetVerticalScrollbarEnabled(void)
+{
+	// IM-2014-08-25: [[ Bug 13272 ]] Show / hide scrollbars by setting the overflow style to empty / "hidden".
+	return !GetOverflowHidden(kMCCefScrollbarVertical);
+}
+
+void MCCefBrowserBase::SetVerticalScrollbarEnabled(bool p_scrollbars)
 {
 	// IM-2014-08-25: [[ Bug 13272 ]] Show / hide scrollbars by setting the overflow style to empty / "hidden".
-	return !GetOverflowHidden();
+	/* UNCHECKED */ SetOverflowHidden(kMCCefScrollbarVertical, !p_scrollbars);
```

Comment:
If `SetOverflowHidden()` can't ever fail, why does it return a flag indicating whether it succeeded?  

We're going to have to deal with the problem of communicating error information from native code in extensions at some point! @runrevmark, what do you think?